### ISSUE TITLE
[Feature Request] Add LoaderOptions support for nifti-volume-loader

### ISSUE THREAD

**Issue by frank-msft** [cornerstonejs/cornerstone3D#1403]
### What feature or change would you like to see made?

dicom-image-loader supports LoaderOptions, which allow the application to specify HTTP headers for authentication.  This feature request is to add LoaderOptions to nifti-volume-loader to support authentication.

### Why should we prioritize this feature?

Without HTTP header support for authentication, nifti-volume-loader will not be able to load images from cloud storage.

**Comment by sedghi** [comment]
I agree that we should add it. Could you take the lead on this? Since we don't use Cornerstone WADO for loading, perhaps the `cornerstone.init` function should accept new parameters like `beforeSend` or something similar to what we do in the WADO loader.
